flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlNodeToOperationConversion.java

```diff
@@ -579,22 +578,18 @@ private Operation convertCreateDatabase(SqlCreateDatabase sqlCreateDatabase) {
         String databaseName =
                 (fullDatabaseName.length == 1) ? fullDatabaseName[0] : fullDatabaseName[1];
         boolean ignoreIfExists = sqlCreateDatabase.isIfNotExists();
-        String databaseComment =
-                sqlCreateDatabase
-                        .getComment()
-                        .map(comment -> comment.getValueAs(NlsString.class).getValue())
-                        .orElse(null);
+        String databaseComment = OperationConverterUtils.getComment(sqlCreateDatabase.getComment());
         // set with properties
         final Map<String, String> properties =
-                OperationConverterUtils.getProperties(sqlCreateDatabase.getPropertyList());
+                OperationConverterUtils.getProperties(sqlCreateDatabase.getProperties());
```

Comment:
also here let's return a Map<String, String> from the parser. We shouldn't expose parser internals to the caller.
```suggestion
                sqlCreateDatabase.getProperties();
```

Reply:
makes sense, added same also for `List<String>`